internal/oci/runtime_oci.go

```diff
@@ -409,7 +409,7 @@ func (r *runtimeOCI) ExecContainer(ctx context.Context, c *Container, cmd []stri
 
 	args := r.defaultRuntimeArgs()
 	args = append(args, "exec", "--process", processFile, c.ID())
-	execCmd := cmdrunner.Command(r.handler.RuntimePath, args...) // nolint: gosec
+	execCmd := cmdrunner.CommandContext(ctx, r.handler.RuntimePath, args...) // nolint: gosec
 	if v, found := os.LookupEnv("XDG_RUNTIME_DIR"); found {
 		execCmd.Env = append(execCmd.Env, fmt.Sprintf("XDG_RUNTIME_DIR=%s", v))
```

Comment:
Because the pty may not drain io, we may want `exec.WaitDelay`. Should since this needs a duration, should I defer to a separate PR?

Reply:
I think including it here makes sense to me. we could either hardcode it, or set it to the container's stop timeout (though that could be arbitrarily long which could defeat the purpose of this PR)

Reply:
For now I can hardcode it.